include/eve/algo/views/iota.hpp

```diff
@@ -66,46 +66,46 @@ namespace eve::algo::views
     static N iterator_cardinal() { return {}; }
 
     template <typename _N>
-    auto cardinal_cast(_N) const { return base_plus_offset_iterator<T, _N>{base, scale, i}; }
+    auto cardinal_cast(_N) const { return base_plus_offset_iterator<T, _N>{base, step, i}; }
 
     base_plus_offset_iterator& operator+=(std::ptrdiff_t n)
     {
       i += n;
-      wide_cur += scale * n;
+      wide_cur += step * n;
       return *this;
     }
 
     friend std::ptrdiff_t operator-(base_plus_offset_iterator x, base_plus_offset_iterator y)
     {
       EVE_ASSERT(x.base == y.base,   "different base");
-      EVE_ASSERT(x.scale == y.scale, "different scale");
+      EVE_ASSERT(x.step == y.step, "different step");
```

Comment:
please align with the top one if this one is changed.
Same for other assertions.

Reply:
ok I'll do that.
For the alignment too, but remember that before release we intend to pass all files to clang-format-14

Reply:
We didn't set up clang format yet, did we?

@jfalcou

Reply:
we need to complete it. The last patch in clangformat14 fixed the egregious support for if constexpr. There is still
hick ups on requires clause but they are manageable.

We need to home in a final setup then proceed by part to convert existing file to new format.